profiler/appengine/flexible/app.yaml

```diff
@@ -0,0 +1,17 @@
+runtime: python
+env: flex
+entrypoint: gunicorn -b :$PORT main:app
+
+runtime_config:
+  python_version: 3
```

Comment:
It would be helpful to add a comment here that using python_version 3, Python 3.7.x is used which supports both Wall & CPU profiling. 
https://cloud.google.com/appengine/docs/flexible/python/runtime

Reply:
Done.